__tests__/rpcProvider.test.ts

```diff
@@ -71,6 +71,11 @@ describeIfRpc('RPCProvider', () => {
     expect(typeof blockNumber).toBe('number');
   });
 
+  test('getGasprice', async () => {
+    const gasPrice = await rpcProvider.getGasPrice(967555);
```

Comment:
This test cannot work on devnet...

24: Block not found: undefined

Reply:
"latest" as parameter should be OK.